modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -51,6 +51,7 @@ namespace detail
         CV_STRING,     // std::string user G-API data
         CV_POINT,      // cv::Point user G-API data
         CV_POINT2F,    // cv::Point2f user G-API data
+        CV_POINT3F,    // cv::Point3f user G-API data
```

Comment:
Missed the below type? See GOpaqueTraits

Reply:
I didn't exposed it for purpose, because there weren't such cases

Reply:
But it is an inconsistency now, isnt it?

And now its only you know the reason for it (ok, and me)

Reply:
Done